app/controllers/component_preview_controller.rb

```diff
@@ -1,9 +1,15 @@
-class ComponentPreviewController < ViewComponentsController
-  include ActionView::Helpers::AssetTagHelper
-  helper Lookbook::PreviewHelper
-  include Lookbook::PreviewController
-  include ScriptHelper
+if IdentityConfig.store.component_previews_enabled
```

Comment:
I think `if` and `else` bodies are reversed here?

Reply:
outdated as of 5031d8932e8b66ead5a274514861434c8af2fc47